debian/changelog

```diff
@@ -1,3 +1,19 @@
+cloud-init (22.4.2-0ubuntu0~22.10.1) kinetic; urgency=medium
+
+  * Upstream snapshot based on 22.4.2 upstream release. (LP: #1997559)
+    - List of changes from upstream can be found at
+      https://raw.githubusercontent.com/canonical/cloud-init/22.4.2/ChangeLog
+
+ -- James Falcon <james.falcon@canonical.com>  Mon, 28 Nov 2022 10:45:45 -0600
+
+cloud-init (22.4.1-0ubuntu0~22.10.1) kinetic; urgency=medium
+
+  * Upstream snapshot based on 22.4.1 upstream release. (LP: #1844191)
```

Comment:
I think this bug should be LP: #1996645 for both of these d/changelog entries (22.4.2 and 22.4.1)  since we have an SRU process bug. Because we didn't officially release beyond -proposed. We don't need a separate process LP bug for 22.4.2

Reply:
Because of our SRU exception, we don't generally track individual bugs in d/changelog but rather our SRU process bug LP: #1996645. This bug should be reflected in both 22.4.1 and 22.4.2 or just in 22.4.2, but it's your call. Maybe in this case (because of regression detection during SRU verification it is worth itemizing the unique Bugs fixed in this release).

Here's a diff that aligns with our upstream publish in lunar that captures both our SRU process bug and the unique bugs fixed for which we can perform supplemental validation if needed.

```diff
 diff --git a/debian/changelog b/debian/changelog
index 5f2c734d..28e24c2a 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -1,16 +1,19 @@
 cloud-init (22.4.2-0ubuntu0~22.04.1) jammy; urgency=medium
 
-  * Upstream snapshot based on 22.4.2 upstream release. (LP: #1997559)
+  * Upstream snapshot based on 22.4.2 upstream release. (LP: #1996645)
     - List of changes from upstream can be found at
       https://raw.githubusercontent.com/canonical/cloud-init/22.4.2/ChangeLog
+    - Bugs fixed in this snapshot: (LP: #1997559)
+
 
  -- James Falcon <james.falcon@canonical.com>  Mon, 28 Nov 2022 10:32:24 -0600
 
 cloud-init (22.4.1-0ubuntu0~22.04.1) jammy; urgency=medium
 
-  * Upstream snapshot based on 22.4.1 upstream release. (LP: #1844191)
+  * Upstream snapshot based on 22.4.1 upstream release. (LP: #1996645)
     - List of changes from upstream can be found at
       https://raw.githubusercontent.com/canonical/cloud-init/22.4.1/ChangeLog
+    - Bugs fixed in this snapshot: (LP: #1844191)
 
  -- James Falcon <james.falcon@canonical.com>  Fri, 18 Nov 2022 20:22:28 -0600
 
```